src/services/redisSessionStorage.ts

```diff
@@ -214,9 +249,30 @@ export default class RedisSessionStorage extends RedisStorageUtils
       );
     });
 
+    const deleteMyPortalToken = new Promise<Either<Error, true>>(resolve => {
```

Comment:
in every part where you have the same operation on session/wallet/myportal token would have been better to `sequenceT`()  some `TaskEithers` into an object which keys are the ones from `RequiredUserTokensV2`. in this way when you add a new token into `RequiredUserTokensV2` you automatically have an hint from the compiler on where the new token must be handled

Reply:
Created a new Story in [here](https://www.pivotaltracker.com/story/show/174730998)